### PR TITLE
New Post: Announcing Alchemy Conf

### PR DESCRIPTION
**Pull request by frm**
This is ghost written by me, with permission and thumbs up from Roberto.
It was already reviewed by him.

### CODE REVIEW

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+that, in retrospect, defined us. We were presented with the opportunity to
+support a small Ruby conference called Ruby Norte and decided to bite the
+bullet. Sponsorship was "only” 250€. The use of quotes is not an exaggeration
+since, at the time, when we stared at our bank account statement, we saw 200€
+staring back at us. The remaining 50€ were put out of our own pockets. I can
+never give a correct answer to the question of "how important the community is,”
+but **I can look back and proudly say it has always been a part of us.**
+
+A lot more followed. **We started organizing several meetups** (Braga.rb,
+Braga.js, Braga.Design, Braga.Product, Braga BEAM), **helped others by hosting
```
Comment (zamith) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:29]:
```suggestion
Braga.js, Braga.Design, Braga.Product, Braga BEAM), helped others by hosting
```

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+them in our office, eventually went around to organize [Ruby Conf
+Portugal][rubyconf] from 2014 to 2016. From 2016 to 2018, we also had [Mirror
+Conf][mirrorconf]**, a conference that aimed to become a conversation between
+designers and front-end developers. Doing this for years on end took its toll,
+of course, and we needed a break.
+
+We have evolved from working mainly as a Ruby on Rails consultancy firm to a
+venture studio during the past few years, where we help create new startups.
+With this transition, we found Elixir to be a valuable alternative that we
+should invest, and we’ve been working with it for more than four years. Many of
```
Comment (zamith) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:39]:
```suggestion
should invest in, and we’ve been working with it for more than four years. Many of
```

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+discounts, so if you are a student, send us an email to [hello@alchemyconf.com].
+
+Like the previous conferences we organized, Alchemy Conf is community-driven.
+This means it is a non-profit. Our focus is and always has been to inspire the
+community. To create an opportunity for developers and enthusiasts to get
+together, to share and learn, experience, and network. From the attendees to the
+speakers, we want everybody to feel like Alchemy Conf is a little bit of their
+own. This is why we want to fully commit to transparency. **We will be making a
+transparency report containing the expenses and earnings of the conference. Any
+profit that may arise will be split equally amongst the speakers.**
```
Comment (zamith) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:69]:
```suggestion
profit that may arise will be given back to the community.**
```

This gives us more wiggle room, keeping the assurance that it's a not for profit event.

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+## The Road to Alchemy Conf
+
+Even though we’re almost seven months away from the start of the conference, we
+want everyone to feel like they are a part of it. We want the road to Alchemy
+Conf to be a journey. And **we want to bring closer the community during times of
```
Comment (zamith) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:75]:
```suggestion
Conf to be a journey. And **we want to bring the community closer during times of
```

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+hardship, even though we can’t all be physically together yet.**
+
+We will be setting up **three online workshops in the months leading up to the
+conference, all of them around Elixir development, to be given by leading
+developers in the community. Each workshop will have a cost of 50€. Taking part
+in all three will earn you a free ticket to the conference.** We’re aiming to
+encourage and reward taking part in community events. That’s why the total cost
+of the lot, 150€, is the same as an early bird ticket - they won’t be making any
```
Comment (zamith) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:83]:
```suggestion
of the lot, 150€, is the same as an early bird ticket - there won't be any
```

File: src/posts/2020-09-11-announcing-alchemy-conf.md (hunk 1 above)
Comment (frm) [reply src/posts/2020-09-11-announcing-alchemy-conf.md:29]: The end of this highlight comes after "Mirror Conf", a few lines below. [Deploy preview here](https://deploy-preview-385--subvisual.netlify.app/blog/posts/announcing-alchemy-conf/). Is it too much though?

File: src/posts/2020-09-11-announcing-alchemy-conf.md (hunk 3 above)
Comment (frm) [reply src/posts/2020-09-11-announcing-alchemy-conf.md:69]: Adding your suggestion, but I'd be more comfortable if we were upfront with what we are doing. I think we should finish the discussion on this topic on Basecamp, before the announcement is published. To make sure everyone is on board.

**Review by zamith — APPROVED** [review]

File: src/posts/2020-09-11-announcing-alchemy-conf.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
+---
+path: /posts/announcing-alchemy-conf/
+title: "Announcing Alchemy Conf"
+author: roberto-machado
```
Comment (naps62) [inline src/posts/2020-09-11-announcing-alchemy-conf.md:4]: out of curiosity, why did you feel the need to be a ghost writer, and not put the post under your own name?

Comment (frm) [reply src/posts/2020-09-11-announcing-alchemy-conf.md:4]: Well, it's always been a CEO thing and Roberto's been around for longer and command the conference. He's also been the force to push us over the edge when we were considering moving forward or not. It's only fair.

**Pull request merged**